src/llmq/utils.cpp

```diff
@@ -58,6 +63,31 @@ void PreComputeQuorumMembers(const CBlockIndex* pQuorumBaseBlockIndex, bool rese
     }
 }
 
+uint256 GetHashModifier(Consensus::LLMQType llmqType, const CBlockIndex* pQuorumBaseBlockIndex)
```

Comment:
I think we should make this param a const ref LLMQParams to avoid calling GetParams since the callers have params